packages/e2e-tests/test/e2e-tls.spec.ts

```diff
@@ -241,13 +241,13 @@ describe('e2e TLS', function () {
         expect(prompt.state).to.equal('prompt');
 
         const logPath = path.join(logBasePath, `${shell.logId}_log`);
-        const logContents = await readReplLogfile(logPath);
+        const logContents = await readReplLogFile(logPath);
         expect(
-          logContents.find((line) => line.id === 1_000_000_049).attr
+          logContents.find((line) => line.id.__value === 1_000_000_049)?.attr
```

Comment:
Looks like CI doesn't like this modification, and I also can't see where this change is coming from